draft-ietf-quic-http.md

```diff
@@ -1463,6 +1463,13 @@ could pose a security risk to an incautious implementer.  An implementation MUST
 ensure that the length of a frame exactly matches the length of the fields it
 contains.
 
+Certain HTTP implementations use the client address for logging or
```

Comment:
Can you add a sub-heading(##) for each of these considerations, like in the transport draft?

Reply:
Yes, but I think I'd rather do that as a separate organization PR and keep this one focused on this issue.